feature_engine/selection/drop_psi_features.py

```diff
@@ -247,8 +252,13 @@ def __init__(
         if not isinstance(switch, bool):
             raise ValueError(f"switch must be a boolean. Got {switch} instead.")
 
-        if not isinstance(threshold, (float, int)) or threshold < 0:
-            raise ValueError(f"threshold must be >= 0. Got {threshold} instead.")
+        if (
+            (isinstance(threshold, str) and (threshold != 'auto')) or
+            (isinstance(threshold, (float, int)) and threshold < 0)
+        ):
+            raise ValueError(
+                f"threshold must be positive float or 'auto'. Got {threshold} instead."
```

Comment:
small tweak:

threshold must be **a** positive float ...

or maybe better:

threshold must be greater than 0 or ...

because it could also be an int as per our definition